README.md

```diff
@@ -115,3 +115,78 @@ if (handleStartupEvent()) {
   return;
 }
 ```
+
+## Checking for updates, downloading and installing
```

Comment:
I believe this code now exists in Electron and is available via the `autoUpdater` API so I don't think we need to document a customer version here.

https://github.com/atom/electron/pull/1984

https://github.com/atom/electron/blob/master/docs/api/auto-updater.md

Am I missing something?